clr-k8s-examples/create_stack.sh

```diff
@@ -279,7 +289,310 @@ function set_repo_version() {
 	popd
 
 }
+function detect_k8s() {
+   if [[ -z "$K8S_VER" ]]; then
+		 K8S_VER=$(kubectl version | grep Server | grep -Po 'GitVersion:"(.*?)",' | grep -Po '"(.*?)"' | tr -d '"')
+	 fi
+}
+detect_k8s
+declare -A command_handlers#!/usr/bin/env bash
```

Comment:
+ ./create_stack.sh all
./create_stack.sh: line 298: declare: `command_handlers#!/usr/bin/env': not a valid identifier

Reply:
Thanks @hnanni It is still a work in progress but almost there.

Reply:
@hnanni Updated and removed that call (which was an error) but still working on it overall.